### PR TITLE
Add test and Docs for angle property on Attached Components class

### PR DESCRIPTION
**Pull request by joaofoes**

### CODE REVIEW

**Comment by joaofoes** [comment]
Ahm, don't quite know where the test failed.

**Comment by rafaellehmkuhl** [comment]
From travis:

```python
def test_angle(attached_component, freebody_component):
        attached_component.set_parent(freebody_component)
        angle = math.degrees(attached_component.angle)
        assert(angle == 14)
        attached_component.actuation_angle = math.radians(10)
        angle = math.degrees(attached_component.angle)
>       assert(angle == 24)
E       assert 24.000000000000004 == 24
```

It's just a float precision problem. Can you substitute the ```assert``` function for the ```numpy.testing.assert_almost_equal()```?

File: docs/Components/AttachedComponent.md [hunk 1]
```diff
@@ -59,4 +59,51 @@ print(math.degrees(aileron.angle))
 plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
+```
+##Angle:
+The property angle will return to you the angle that the given component is set.
+
+Let's say we have a component 'Wing' attached to the body of the plane, that it's set as a FreeBody Component called 'Plane'. The property angle will take in count the plane angle and the relative angle of the wing, just as shown in the example below:
```
Comment (rafaellehmkuhl) [inline docs/Components/AttachedComponent.md:66]: typo: [...] that is set as [...]

File: docs/Components/AttachedComponent.md [hunk 2]
```diff
@@ -59,4 +59,51 @@ print(math.degrees(aileron.angle))
...
+    name='wing',
+    type='wing',
+    mass=0.3,
+    relative_position=Vector2(x=-0.15, y=0.2)
+)
+
+plane.angle = math.radians(3.2)
+wing.relative_angle = math.radians(1)
+print(math.degrees(wing.angle))
+>>> 4.2
```
Comment (rafaellehmkuhl) [inline docs/Components/AttachedComponent.md:89]: Despite being possible to set the "relative_angle" property after the component instantiation, we are setting as a convention to always set it on instantiation, together with relative_position.

In the future we are going to make this mandatory, so the user always set it and the library don't face calculation problems.

File: docs/Components/AttachedComponent.md [hunk 3]
```diff
@@ -59,4 +59,51 @@ print(math.degrees(aileron.angle))
...
+    type='aileron',
+    mass=0.02,
+    relative_position=Vector2(x=-0.3, y=0),
+    relative_angle=math.radians(0),
+)
+
+aileron.actuation_angle(math.radians(15))
+aileron.set_parent(wing)
+print(math.degrees(aileron.angle))
+>>> 19.2
```
Comment (rafaellehmkuhl) [inline docs/Components/AttachedComponent.md:108]: Nice catch!

File: tests/Components/test_AttachedComponent.py [hunk 4]
```diff
@@ -1,4 +1,6 @@
 from adr.Components import AttachedComponent
+from adr.Components import FreeBody
```
Comment (rafaellehmkuhl) [inline tests/Components/test_AttachedComponent.py:2]: Import those on the same line.

**Review by rafaellehmkuhl — CHANGES_REQUESTED** [review]

**Review by CassianoMontibeller — APPROVED** [review]

**Review by rafaellehmkuhl — APPROVED** [review]

**Pull request merged**